app/models/user.rb

```diff
@@ -1,6 +1,14 @@
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :role, presence: true, inclusion: { in: %w(member admin)}
+
+  def admin?
+    role == "admin"
+  end
```

Comment:
normally I wouldnt allow addition of an admin role just yet, because there is no associated feature, but I think this is passable for now